CODE_OF_CONDUCT.md

```diff
@@ -7,7 +7,7 @@ The xi-editor project follows the Rust Code of Conduct, reproduced below.
 **Contact**: [mods@xi-editor.io](mailto:mods@xi-editor.io)
 
 * We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other similar characteristic.
-* On IRC, please avoid using overtly sexual nicknames or other nicknames that might detract from a friendly, safe and welcoming environment for all.
+* On Zulip, please avoid using overtly sexual nicknames or other nicknames that might detract from a friendly, safe and welcoming environment for all.
```

Comment:
I wonder if it just makes sense to say something like, "On chat platforms (like Zulip or IRC).."